src/Common/DomainResultOfT.cs

```diff
@@ -142,55 +155,67 @@ public static IDomainResult<TValue> From<TOld>(IDomainResult<TOld> result)
+		public static Task<IDomainResult<TValue>> UnauthorizedTask(string? message = null) => Task.FromResult(Unauthorized(message));
 
 		/// <summary>
 		///		Returns <see cref="DomainOperationStatus.Conflict"/> status wrapped in a <see cref="Task{T}"/>. Gets converted to HTTP code 409 (Conflict)
 		/// </summary>
 		/// <param name="message"> Optional message </param>
-		public static Task<IDomainResult<TValue>> ConflictTask(string? message = null)	=> Task.FromResult(Conflict(message));
+		public static Task<IDomainResult<TValue>> ConflictTask(string? message = null) => Task.FromResult(Conflict(message));
 
 		/// <summary>
 		///		Returns <see cref="DomainOperationStatus.Failed"/> status wrapped in a <see cref="Task{T}"/>. Gets converted to HTTP code 400/422
 		/// </summary>
 		/// <param name="error"> Optional message </param>
-		public static Task<IDomainResult<TValue>> FailedTask(string? error = null)			=> Task.FromResult(Failed(error));
+		public static Task<IDomainResult<TValue>> FailedTask(string? error = null) => Task.FromResult(Failed(error));
+
 		/// <summary>
 		///		Returns <see cref="DomainOperationStatus.Failed"/> status wrapped in a <see cref="Task{T}"/>. Gets converted to HTTP code 400/422
 		/// </summary>
 		/// <param name="errors"> Custom messages </param>
-		public static Task<IDomainResult<TValue>> FailedTask(IEnumerable<string> errors)	=> Task.FromResult(Failed(errors));
+		public static Task<IDomainResult<TValue>> FailedTask(IEnumerable<string> errors) => Task.FromResult(Failed(errors));
+
 		/// <summary>
 		///		Returns <see cref="DomainOperationStatus.Failed"/> status wrapped in a <see cref="Task{T}"/>. Gets converted to HTTP code 400/422
 		/// </summary>
 		/// <param name="validationResults"> Results of a validation request </param>
-		public static Task<IDomainResult<TValue>> FailedTask(IEnumerable<ValidationResult> validationResults)
-																							=> Task.FromResult(Failed(validationResults));
+		public static Task<IDomainResult<TValue>> FailedTask(IEnumerable<ValidationResult> validationResults) =>
+			Task.FromResult(Failed(validationResults));
 
 		/// <summary>
 		///		Returns <see cref="DomainOperationStatus.CriticalDependencyError"/> (failed dependency call) status wrapped in a <see cref="Task{T}"/>. Gets converted to HTTP code 503 (Service Unavailable)
 		/// </summary>
 		/// <param name="error"> Optional error message </param>
-		public static Task<IDomainResult<TValue>> CriticalDependencyErrorTask(string? error = null)	
-																							=> Task.FromResult(CriticalDependencyError(error));
-		
+		public static Task<IDomainResult<TValue>> CriticalDependencyErrorTask(string? error = null) =>
+			Task.FromResult(CriticalDependencyError(error));
+
 		#endregion // Extensions of 'Task<IDomainResult<T>>' [STATIC, PUBLIC] -
+
+		public static implicit operator DomainResult<TValue>(DomainResult result)
```

Comment:
Don't like that it leads to a runtime exception if a dev forgets to check `IsSuccess` prior to an implicit conversion. Though, it can be easily mitigated by changing the method's body to calling the constructor (with implicit `default` for the value):
```csharp
public static implicit operator DomainResult<TValue>(DomainResult domainResult) => new (domainResult);
```